lib/src/chatbox.dart

```diff
@@ -262,11 +265,11 @@ class ChatBoxState extends State<ChatBox> {
 
     return InAppWebView(
       initialSettings: InAppWebViewSettings(
-        useHybridComposition: true,
-        disableInputAccessoryView: true,
-        transparentBackground: true,
-        useShouldOverrideUrlLoading: true,
-      ),
+          useHybridComposition: true,
+          disableInputAccessoryView: true,
+          transparentBackground: true,
+          useShouldOverrideUrlLoading: true,
+          applicationNameForUserAgent: "TalkJS_Flutter/0.13.0"),
```

Comment:
My biggest fear is that we forget to update this manually.

We need to find a way of automating this.
Given that it's not possible to read the `pubspec.yaml` file runtime, I propose that re put a placeholder here, and in the build script we change this placeholder with the correct version number read from `pubspec.yaml`